src/main/java/org/mskcc/cmo/messaging/model/PatientMetadata.java

```diff
@@ -0,0 +1,78 @@
+package org.mskcc.cmo.messaging.model;
+
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.UUID;
+import org.neo4j.ogm.annotation.GeneratedValue;
+import org.neo4j.ogm.annotation.Id;
+import org.neo4j.ogm.annotation.NodeEntity;
+import org.neo4j.ogm.annotation.Relationship;
+import org.neo4j.ogm.annotation.typeconversion.Convert;
+import org.neo4j.ogm.id.UuidStrategy;
+import org.neo4j.ogm.typeconversion.UuidStringConverter;
+
+/**
+ *
+ * @author ochoaa
+ */
+@NodeEntity
+public class PatientMetadata implements Serializable {
+    @Id @GeneratedValue(strategy = UuidStrategy.class)
+    @Convert(UuidStringConverter.class)
+    private UUID metaDbUuid;
+    private String investigatorPatientId;
+    @Relationship(type="PX_TO_SP", direction=Relationship.OUTGOING)
+    private List<SampleMetadataEntity> sampleMetadataList;
+    @Relationship(type="PX_TO_PX", direction=Relationship.INCOMING)
+    private List<LinkedPatient>  linkedPatientList;
+
+    public PatientMetadata() {}
+
+    public UUID getMetaDbUuid() {
+        return metaDbUuid;
+    }
+
+    public void setMetaDbUuid(UUID metaDbUuid) {
```

Comment:
Why does PatientMetadata contain methods to get specific patient aliases?  Shouldn't this only be on LinkedPatient instances?

Reply:
see comment [here](https://github.com/mskcc/cmo-messaging-java/pull/1/files?diff=unified&w=1#r524556713)

Reply:
it probably *isn't* that necessary to have all of the possible relationships defined but this is the approach I went with for now since it's less effort to remove code/methods than to add them in